docs/ui/roles.md

```diff
@@ -0,0 +1,138 @@
+
+The built-in roles will serve the needs of most users, but your team may need to configure custom roles, giving users access to specific permissions within a workspace.
+
+Custom roles can inherit permissions from a built-in role. This enables tweaks to meet your organization’s needs, while ensuring users can still benefit from Prefect’s default workspace role permission curation as new functionality becomes available.
+
+Custom workspace roles can also be created independent of Prefect’s built-in roles. This option gives workspace admins full control of user access to workspace functionality. However, for non-inherited custom roles, the workspace admin takes on the responsibility for monitoring and setting permisssions for new functionality as it is released.
+
+See [Role permissions](#role-permissions) for details of permissions you may set for custom roles.
+
+After you create a new role, it become available in the organization **Members** page and the **Workspace Sharing** page for you to apply to users.
+
+### Inherited roles
+
+A custom role may be configured as an **Inherited Role**. Using an inherited role allows you to create a custom role using a set of initial permissions associated with a built-in Prefect role. Additional permissions can be added to the custom role. Permissions included in the inherited role cannot be removed.
+
+Custom roles created using an inherited role will follow Prefect's default workspace role permission curation as new functionality becomes available.
+
+To configure an inherited role when configuring a custom role, select the **Inherit permission from a default role** check box, then select the role from which the new role should inherit permissions.
+
+![Creating a custom role for a workspace using inherited permissions in Prefect Cloud](/img/ui/org-inherited-role.png)
+
+## Workspace role permissions
+
+The following permissions are available for custom roles.
+
+### Blocks
+
+| Permission | Description |
+| --- | --- |
+| View blocks | User can see configured blocks within a workspace. |
+| Create, edit, and delete blocks | User can create, edit, and delete blocks within a workspace. Includes permissions of **View blocks**. |
+
+### Concurrency
+
+| Permission | Description |
+| --- | --- |
+| View concurrency limits | User can see configured concurrency limits within a workspace. |		
+| Create, edit, and delete concurrency limits | User can create, edit, and delete concurrency limits within a workspace. Includes permissions of **View concurrency limits**. |		
+
+### Deployments
+
+| Permission | Description |
+| --- | --- |
+| View deployments | User can see configured deployments within a workspace. |
+| Run a deployment | User can run a deployments within a workspace. This does not give a user permission to execute a flow. This only gives a user (view their key) the ability to run a deployment, so another user/key must actually execute that flow. Includes permissions of **View deployments**. |
```

Comment:
```suggestion
| Run deployments | User can run deployments within a workspace. This does not give a user permission to execute the flow associated with the deployment. This only gives a user (view their key) the ability to run a deployment, so another user/key must actually execute that flow. Includes permissions of **View deployments**. |
```

Reply:
This is a great point. I think my only clarification would be that the actual execution could be either a service account, or another user's API key. Like you say, regardless either of whom would need "Create, update, and run flows" permission to actually execute the requested run.

Reply:
@nathanielaldon see the latest change to this text for my take, which I think captures the nuance

Reply:
That feels right to me! Thanks a ton for looking at it!